src/firefly/js/charts/ChartUtil.js

```diff
@@ -579,9 +579,9 @@ function updateChartData(chartId, traceNum, tablesource, action={}) {
     } else {
         if (!isFullyLoaded(tbl_id)) return;
         const tableModel = getTblById(tbl_id);
-        if (!getColValStats(tbl_id)) {
-            dispatchLoadTblStats(tableModel.request);
-        }
+
+        // may not need to call it as often.  if this becomes a performance issue, then optimize.
+        dispatchLoadTblStats(tableModel.request);
```

Comment:
not sure why condition was removed. Do we want to recompute table statistics because new column was added?

Reply:
Yes, it's for when columns are added/removed. There is logic in dispatchLoadTblStats that determine when to load.